Beecrowd.Beginner.Solutions/1036-BashkarasFormula.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Beecrowd
+{
+    class BhaskarasFormula
+    {
+        public static void Run()
+        {
+            string[] inputs = Console.ReadLine().Split(' ');
+            double a = Convert.ToDouble(inputs[0]);
+            double b = Convert.ToDouble(inputs[1]);
+            double c = Convert.ToDouble(inputs[2]);
+
+            if (a == 0)
+            {
+                Console.WriteLine("Impossivel calcular");
+                return;
+            }
+
+            double difference = (b * b) - (4 * a * c);
```

Comment:
You can move this part of your code up to your first if condition and check it in your first if condition as well, you don't need to add a separate if for it.

Reply:
Fixed